recipe/meta.yaml

```diff
@@ -60,10 +60,11 @@ requirements:
     - gmp  # [unix]
     - libva  # [linux64]
     - aom
-    - svt-av1
+    - svt-av1  1.4.1   # latest version to support ffmpeg 5.0
```

Comment:
Where is that constraint coming from?

The upstream [docs](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Build-Guide.md#svt-av1-ffmpeg-plugin-installation) I could find don't mention a cap (or lower bound), and the [svt-av1](https://github.com/conda-forge/svt-av1-feedstock/blob/main/recipe/meta.yaml) recipe doesn't mention ffmpeg either.

Reply:
From my previous attempts:
https://github.com/conda-forge/ffmpeg-feedstock/pull/196